src/main/java/org/wso2/carbon/module/swiftiso20022/JsonToMT940Transformer.java

```diff
@@ -85,121 +83,29 @@ public void connect(MessageContext messageContext) throws ConnectException {
      * Method to validate the request payload.
      *
      * @param requestPayload  Request payload
-     * @return               Error model if there is an error, else empty error model
+     * @return               Validation Result if there is an error, else empty Validation Result
      */
-    private ErrorModel validateRequestPayload(RequestPayloadModel requestPayload) {
+    private ValidationResult validateRequestPayload(String requestPayload) {
```

Comment:
Can't we go with something like this here?

```
JsonToMT940PayloadValidator.getMT940ValidationEngine(requestPayloadJson)
    .addValidateBalances()
    .addValidateTransactionDetails()
    .validate();
```

Reply:
Balances and traansactions are fields specific only to MT940. So we cannot have common methods to those